src/pages/account/settings/components/PhoneView.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment, PureComponent } from 'react';
+
+import { Input } from 'antd';
+import styles from './PhoneView.less';
+
+interface PhoneViewProps {
+  value?: string;
+  onChange?: (value: string) => void;
+}
+
+class PhoneView extends PureComponent<PhoneViewProps> {
+  render() {
+    const { value, onChange } = this.props;
+    let values = ['', ''];
+    if (value) {
+      values = value.split('-');
+    }
+    return (
+      <Fragment>
+        <Input
+          className={styles.area_code}
+          value={values[0]}
+          onChange={e => {
+            if (onChange) {
+              onChange(`${e.target.value}-${values[1]}`);
+            }
+          }}
+        />
+        <Input
+          className={styles.phone_number}
+          onChange={e => {
+            if (onChange) {
+              onChange(`${values[0]}-${e.target.value}`);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)